Makefile

```diff
@@ -5,7 +5,7 @@ build-native:
 
 build-bs:
 	@printf "\n\e[31mBuilding bs ...\e[0m\n"
-	cd bs && npm run build
+	cd bucklescript && npm run build
```

Comment:
If we flattened the directory structure, we shouldn't need `cd bucklescript` I would think?

Reply:
We don't, I didn't notice since both npm and dune don't mind being run from a sub-directory